### PR TITLE
[🧹 UNIFICATION CLEANUP] Refactor `admin.extensions.v1/components/application`

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

- Remove API resource v1 related extensions
- Move Try It Application related logic to `admin.applications.v1`

### Related Issues

- https://github.com/wso2/product-is/issues/20802

### Related PRs

- N/A

### Checklist
- [ ] e2e cypress tests locally verified. (for internal contributers) 
- [ ] Manual test round performed and verified. 
- [ ] UX/UI review done on the final implementation. 
- [ ] Documentation provided. (Add links if there are any) 
- [ ] Relevant backend changes deployed and verified 
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any) 
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any) 

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: features/admin.extensions.v1/components/application/constants/try-it-constants.ts [hunk 1]
```diff
@@ -25,12 +25,4 @@ export class TryItApplicationConstants {
      * Application's Display Name.
      */
     public static readonly DISPLAY_NAME = "Asgardeo Try It";
```
Comment (pavinduLakshan) [inline features/admin.extensions.v1/components/application/constants/try-it-constants.ts:27]:
```suggestion
    public static readonly DISPLAY_NAME: string = "Asgardeo Try It";
```

File: features/admin.applications.v1/components/try-it/try-it-create-wizard.tsx [hunk 2]
```diff
@@ -153,7 +151,21 @@ export const LoginPlaygroundWizard: FunctionComponent<LoginPlaygroundWizardProps
                         <Grid.Row textAlign="center">
                             <Grid.Column stretched width={ "16" }>
                                 <GenericIcon
-                                    icon={ InformationIcon }
+                                    icon={ (
+                                        /* eslint-disable max-len */
+                                        <svg width="80" height="80" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/components/try-it/try-it-create-wizard.tsx:156]: Is hardcoding the icon here intentional?

File: features/admin.applications.v1/constants/try-it-constants.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020-2024, WSO2 LLC. (https://www.wso2.com).
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/constants/try-it-constants.ts:2]:
```suggestion
 * Copyright (c) 2022-2024, WSO2 LLC. (https://www.wso2.com).
```

File: features/admin.applications.v1/utils/get-try-it-client-id.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com).
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/utils/get-try-it-client-id.ts:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: features/admin.roles.v2/models/policies.ts [hunk 5]
```diff
@@ -16,5 +16,16 @@
...
+ */
+export enum Policy {
+    /**
+     * Role based policy.
+     */
+    ROLE = "RBAC",
+    /**
+     * No Policy.
+     */
+    NO = "NONE"
```
Comment (pavinduLakshan) [inline features/admin.roles.v2/models/policies.ts:30]: WDYT?
```suggestion
    NONE = "NONE"
```

**Review by pavinduLakshan — COMMENTED** [review]
Let's add changeset as well.

**Review by pavinduLakshan — COMMENTED** [review]
Let's add changeset as well.

**Review by pavinduLakshan — COMMENTED** [review]
Let's add changeset as well.

File: features/admin.applications.v1/components/try-it/try-it-create-wizard.tsx (hunk 2 above)
Comment (brionmario) [reply features/admin.applications.v1/components/try-it/try-it-create-wizard.tsx:156]: Screenshot 2024-08-13 at 10 59 36
This is the icon and there's no point of maintaining this in the `theme` as a re-usable resource.

File: features/admin.roles.v2/models/policies.ts (hunk 5 above)
Comment (brionmario) [reply features/admin.roles.v2/models/policies.ts:30]: This was an already existing model and seems like moving the file and renaming has messed up the Git history.

File: features/admin.applications.v1/utils/get-try-it-client-id.ts (hunk 4 above)
Comment (brionmario) [reply features/admin.applications.v1/utils/get-try-it-client-id.ts]: Fixed with https://github.com/wso2/identity-apps/pull/6744/commits/f9ffd219bfb5ffe6edf6efd22da5bd7728103ec1

File: features/admin.applications.v1/constants/try-it-constants.ts (hunk 3 above)
Comment (brionmario) [reply features/admin.applications.v1/constants/try-it-constants.ts]: Fixed with https://github.com/wso2/identity-apps/pull/6744/commits/208951dab716e32fc593ce386f54686fc1aace70

**Review by pavinduLakshan — APPROVED** [review]

**Review by dasuni-30 — DISMISSED** [review]

**Review by pavinduLakshan — APPROVED** [review]

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**